src/app/migrations/range/range.1.core.txt

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
```

Comment:
Range and Select are not merged yet. Do you plan to merge this PR after the `range` and `select` are merged?

Reply:
You are right they slipped (bad fork) - will merge this PR after I merge the other one.